canonical-web-and-design/ubuntu.com
# Separate engage pages and takeover views

## Done

- Reduce size engage pages https://github.com/canonical-web-and-design/ubuntu.com/issues/10299

These engage pages are most active, so they don't want us to put it in an archive section. Instead we are separating takeovers and engage pages, so the discourse post is reduced about half, this should ease the max limit (180000 characters), although in a year or so might need to be revisited as the number of engage pages grow again.

## QA

- Check the new source of [engage pages](https://discourse.ubuntu.com/t/test-engage-pages/18033) and [takeovers](https://discourse.ubuntu.com/t/test-takeovers/25642). This is the main issue this PR is trying to improve, the code is more complementary.
- Code is basically just refactoring. Moving to views.py with a new instance, since we are now using 2 separate index topics.
- Check https://ubuntu-com-11006.demos.haus/ make sure takeovers work as usual
- Check https://ubuntu-com-11006.demos.haus/takeovers work as usual
- Check https://ubuntu-com-11006.demos.haus/engage work as usual
- On https://ubuntu-com-11006.demos.haus/engage click on a few, fill out the form make sure the whole process works as usual.

## Issue / Card

Fixes #10299 

## Help

[QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html) - [Commit guidelines](https://canonical-web-and-design.github.io/practices/workflow/commit-messages.html)
